panoptes_client/workflow.py

```diff
@@ -179,6 +182,343 @@ def subject_workflow_statuses(self, subject_set_id):
         for status in SubjectWorkflowStatus.where(subject_ids=subject_ids, workflow_id=self.id):
             yield status
 
+    """ CAESAR METHODS """
+
+    def add_to_caesar(self, public_extracts=False, public_reductions=False):
+        """
+        Adds selected Workflow to Caesar. Returns workflow as a dict from Caesar if successful.
+        - **public_extracts** set to True to Enable Public Extracts, Defaults to False
+        - **public_reductions** set to True to Enable Public Reductions. Defaults to False
+
+        Examples::
+            workflow.add_to_caesar()
+            workflow.add_to_caesar(True, True)
```

Comment:
A better example here might be to use named args (`workflow.add_to_caesar(public_extracts=True, public_reductions=True)`). This ex will work but it's kind of confusing, esp since you can pass a single True and get public extracts and private reductions.